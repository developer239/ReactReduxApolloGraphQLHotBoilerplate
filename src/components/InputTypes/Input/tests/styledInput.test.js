import React from 'react'
import { shallow } from 'enzyme'
import { theme } from 'styles'
import { StyledInput } from '../index'
import 'jest-styled-components'


describe('StyledInput Component', () => {
  it('has correct default styles', () => {
    const wrapper = shallow(
      <StyledInput theme={theme}>Button Text</StyledInput>,
    )
    expect(wrapper).toMatchSnapshot()
  })
})
