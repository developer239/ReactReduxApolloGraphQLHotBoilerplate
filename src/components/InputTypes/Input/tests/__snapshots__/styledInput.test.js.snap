// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledInput Component has correct default styles 1`] = `
.c0 {
  padding: 10px 10px;
  border: 1px solid #ccd1d9;
  margin-top: 10px;
}

<styled.input
  theme={
    Object {
      "color": Object {
        "black": "#000",
        "darkGreen": "#37bc9b",
        "darkGrey": "#aab2bd",
        "darkRed": "#da4453",
        "lightGreen": "#48cfad",
        "lightGrey": "#ccd1d9",
        "lightRed": "#ed5565",
        "primary": "#5d9cec",
        "secondary": "#4a89dc",
        "white": "#fff",
      },
      "layout": Object {
        "maxWidth": "62.5rem",
      },
    }
  }
>
  <input
    className="c0"
  />
</styled.input>
`;
